src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -81,6 +87,8 @@ internal class StartHostAction : BaseAction
 
         public string JsonOutputFile { get; set; }
 
+        public string? HostRuntime { get; set; }
```

Comment:
Wondering why this property is nullable; is it okay for HostRuntime to be null?

Reply:
yes! sometimes the user won't pass in the property and we can infer the host runtime by that :)